src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -12,75 +12,60 @@
     <IsPlaceholderTargetFramework Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">true</IsPlaceholderTargetFramework>
-    <EnableDefaultItems>false</EnableDefaultItems>
   </PropertyGroup>
 
-  <!-- On .NET Core, Vector<T> is in System.Private.CoreLib -->
-  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
-    <Compile Include="System\Numerics\ConstantHelper.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>ConstantHelper.tt</DependentUpon>
-    </Compile>
-    <Compile Include="System\Numerics\Register.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Register.tt</DependentUpon>
-    </Compile>
-    <Compile Include="System\Numerics\Vector.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Vector.tt</DependentUpon>
-    </Compile>
-    <Compile Include="System\Numerics\Vector_Operations.cs" />
+  <ItemGroup>
+    <Compile Update="System\Numerics\ConstantHelper.cs"
+             AutoGen="True"
+             DesignTime="True"
+             DependentUpon="ConstantHelper.tt" />
+    <Compile Update="System\Numerics\Register.cs"
+             AutoGen="True"
+             DesignTime="True"
+             DependentUpon="Register.tt" />
+    <Compile Update="System\Numerics\Vector.cs"
+             AutoGen="True"
+             DesignTime="True"
+             DependentUpon="Vector.tt" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
-    <None Include="System\Numerics\GenerationConfig.ttinclude" />
-    <Content Include="System\Numerics\ConstantHelper.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>ConstantHelper.cs</LastGenOutput>
-    </Content>
-    <Content Include="System\Numerics\Register.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>Register.cs</LastGenOutput>
-    </Content>
-    <Content Include="System\Numerics\Vector.tt">
-      <Generator>TextTemplatingFileGenerator</Generator>
-      <LastGenOutput>Vector.cs</LastGenOutput>
-    </Content>
+  <ItemGroup>
+    <None Update="System\Numerics\ConstantHelper.tt"
+          Generator="TextTemplatingFileGenerator"
+          LastGenOutput="ConstantHelper.cs" />
+    <None Update="System\Numerics\Register.tt"
+          Generator="TextTemplatingFileGenerator"
+          LastGenOutput="Register.cs" />
+    <None Update="System\Numerics\Vector.tt"
+          Generator="TextTemplatingFileGenerator"
+          LastGenOutput="Vector.cs" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(IsPlaceholderTargetFramework)' != 'true'">
-    <Compile Include="System\MathF.netstandard.cs" />
-    <Compile Include="System\Numerics\Hashing\HashHelpers.cs" />
-    <Compile Include="System\Runtime\CompilerServices\IntrinsicAttribute.cs" />
```

Comment:
Why are these not getting removed now that the EnableDefaultItems condition has been inverted? Why are they not getting mentioned anymore?

Reply:
The placeholder framework targets file disables default items so those aren't getting imported.